src/components/AuthContainer/AuthContainer.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import AuthContainer, {AuthContainerProps} from '.';
+
+describe('AuthContainer', () => {
+  const baseProps: AuthContainerProps = {
+    children: <div>Children</div>,
+    heading: 'Auth Container Heading',
+  };
+
+  it('renders proper default className', () => {
+    render(<AuthContainer {...baseProps} />);
+    const el = screen.getByTestId('AuthContainer');
+
+    expect(el.className).toBe('AuthContainer');
+  });
+
+  it('renders without error passed in', () => {
+    render(<AuthContainer {...baseProps} />);
+
+    expect(screen.getByTestId('AuthContainer')).toBeTruthy();
+    expect(screen.getAllByText('Auth Container Heading')).toBeTruthy();
+    expect(screen.getAllByText('Children')).toBeTruthy();
```

Comment:
just out of curiosity, any reason you used `getAllByText` instead of `getByText`?

Reply:
Actually, I should have used `getByText` since there's texts are unique. Was copying and pasting from other components 😅

Reply:
I was worried that `getAllByText` will give an empty array `[]` if it didn't find anything, and since `[]` is considered truthy, that this test will give us a false positive. But I checked, and it didn't, so no harm